projects/ui-components/lf-metadata/field-components/field-base-parts/lf-field-base/lf-field-token.service.ts

```diff
@@ -190,7 +190,7 @@ export class LfFieldTokenService implements LfTokenService {
     }
   }
 
-  containsTokenForFieldType(fieldValue: string | null, fieldType: FieldType, isImport: boolean): boolean {
+  containsTokenForFieldType(fieldValue: string, fieldType: FieldType, isImport: boolean): boolean {
```

Comment:
should this still be `fieldValue?: string`?

Reply:
do you mean fieldValue: string | undefined?

Reply:
yeah, same thing.. but I see that you didn't really change any functionality because of the falsy so I'm okay with it